src/core_plugins/console/public/src/es.js

```diff
@@ -18,9 +18,20 @@ module.exports.send = function (method, path, data, server, disable_auth_alert)
   // delayed loading for circular references
   var settings = require("./settings");
 
+  let contentType;
+  if (data) {
+    try {
+      JSON.parse(data);
+      contentType = 'application/json';
```

Comment:
@spalger There is (sort of) an edge case with this check. Consider the following valid `_bulk` API request:

```
POST /foo/bar/_bulk
{ "delete": { "_id": "1" } }
```

The content-type for this request will be set to `application/json`, not `text/plain` as it would be for multiline `_bulk` requests.

I'm okay with this because the body _is_ infact valid JSON in this case. Also, Elasticsearch handles this request with content type as `application/json`. Still, I wanted to point it out in case you feel differently.